### ISSUE TITLE
Waveform disappear 

### ISSUE THREAD

**Issue by Spl4shh** [mixxxdj/mixxx#12404]
### Bug Description

Sometime when i change the music displaying on my deck, the waveform is disappearing

**Comment by foss-** [comment]
Are you able to trigger this issue reliable and if yes, how? Could you attach a screencast of this issue? Which Windows version are you using?

Does this problem also happen, with Mixxx 2.4 beta? https://mixxx.org/download/#beta

**Comment by Spl4shh** [comment]
This issue happen when i switch between 2 musics, but not everytime.
Next time I get this error I will put a screenshot on it 
I use Windows 11
I will try on the beta

Have you ever heard about it ?

**Comment by ronso0** [comment]
I think it'll help already if you described the issue in more detail.
Which waveforms are affected: the scrolling waveform or the overview in the decks?
You load a new track a no waveform shows up, even after the track analysis finished?
Ejecting and loading that same track again does also not give you any waveform?

**Comment by Spl4shh** [comment]
Which waveforms are affected: the scrolling waveform or the overview in the decks?
--> The scrolling one
You load a new track a no waveform shows up, even after the track analysis finished?
--> Even after analysis yes, when I load the track the analysis is already done
Ejecting and loading that same track again does also not give you any waveform?
--> Yes, if a load another track, or the same, the waveform wont ever appear since I restart the app

**Comment by ronso0** [comment]
Oh okay, I get it:
at some point the waveform for one **deck** is cleared, regardless which tracks you load afterwards?

**Comment by Spl4shh** [comment]
Exactly, but it just happen when I load a track, it clear the line concerned

**Comment by ronso0** [comment]
Does it also happen if you explicitly eject a track before loading a new one?

**Comment by Spl4shh** [comment]
I haven't tried this yet, to be honest i never do it

**Comment by mxmilkiib** [comment]
might this still be an issue?

**Comment by acolombier** [comment]
The rendering backend has changed since 2.3, so potentially not an issue anymore. @Spl4shh can you still reproduce the issue on 2.5 or the 2.6 beta?
